glslc/README.asciidoc

```diff
@@ -128,6 +129,20 @@ multiple files generated. A filename of `-` represents standard output.
 
 === Language and Mode Selection Options
 
+[[option-f-auto-bind-uniforms]]
+==== `-fauto-bind-uniforms`
```

Comment:
I'm a little bit unclear about the parameter name. Should we call it `*uniforms` or `*resources`? It feels like to me that `uniform` is from the old OpenGL/GLSL world, while [in Vulkan, resources](https://www.khronos.org/registry/vulkan/specs/1.0-wsi_extensions/html/vkspec.html#descriptorsets) is what used to stand for those data passed in via bindings.

And I really don't like the way they call *everything* as uniforms.